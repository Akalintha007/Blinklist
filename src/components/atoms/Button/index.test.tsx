import React from 'react'
import { render, screen } from '@testing-library/react';
import Button from '.';
// import renderer from 'react-test-renderer'

describe('Testing the ImageComponent', () =>
{it('renders Button', () => {
  render(<Button children="Hello" />);
  const typoElement = screen.getByRole('button');
  expect(typoElement).toMatchSnapshot();
})});

// describe('Testing the ImageComponent', () => {
//   it('render ImageComponent', () => {
//     const image = renderer.create(<Button children="Hello" />).toJSON();
//     expect(image).toMatchSnapshot();
//   });