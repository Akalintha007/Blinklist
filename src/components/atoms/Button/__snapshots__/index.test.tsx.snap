// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the ImageComponent it renders Button 1`] = `
<button
  class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1ctd5mc-MuiButtonBase-root-MuiButton-root"
  tabindex="0"
  type="button"
>
  Hello
  <span
    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
  />
</button>
`;

exports[`Testing the ImageComponent renders Button 1`] = `
<button
  class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1ctd5mc-MuiButtonBase-root-MuiButton-root"
  tabindex="0"
  type="button"
>
  Hello
  <span
    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
  />
</button>
`;

exports[`it renders Button 1`] = `
<button
  class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1ctd5mc-MuiButtonBase-root-MuiButton-root"
  tabindex="0"
  type="button"
>
  Hello
  <span
    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
  />
</button>
`;
